pkg/collector/corechecks/gpu/gpu.go

```diff
@@ -112,10 +112,18 @@ func (m *Check) Run() error {
 	for pid, pidStats := range stats.ProcessStats {
 		// Per-PID metrics are subject to change due to high cardinality
 		tags := []string{fmt.Sprintf("pid:%d", pid)}
-		snd.Gauge(metricNameMemory, float64(pidStats.CurrentMemoryBytes), "", tags)
-		snd.Gauge(metricNameMaxMem, float64(pidStats.MaxMemoryBytes), "", tags)
 		snd.Gauge(metricNameUtil, pidStats.UtilizationPercentage, "", tags)
 
+		currentMem, maxMem := float64(0), float64(0)
+		for _, memStats := range pidStats.Memory {
+			// For now we just sum all the memory types, later we might want to send additional metrics for each type
```

Comment:
I think this comment might be outdated as we only have one metric here, but commenting just in case: [we had a discussion in #gpu-monitoring-squad](https://dd.slack.com/archives/C072J0RR7FB/p1729174993717159) where the consensus seemed to be that it's better to have one metric per type, if we use tags and users are not aware of the type results might be confusing. E.g., if we have `memory{type:global, host:host1} = 2, memory{type:shared, host:host1} = 4`, users that query avg memory by host  would receive 3 instead of 6 which is the memory usage of that host, as they're averaging across all memory types.